bin/ruby-build

```diff
@@ -1,14 +1,16 @@
 #!/usr/bin/env bash
 #
-# Usage: ruby-build [-kvp] <definition> <prefix>
+# Usage: ruby-build [-kpv46] <definition> <prefix>
```

Comment:
Is there a desire to either fix or remove support for those flags?
On Jan 16, 2016 5:56 AM, "Mislav Marohnić" <notifications@github.com> wrote:

> In bin/ruby-build
> <https://github.com/rbenv/ruby-build/pull/879#discussion_r49930410>:
>
> > @@ -1,14 +1,16 @@
> >  #!/usr/bin/env bash
> >  #
> > -# Usage: ruby-build [-kvp] <definition> <prefix>
> > +# Usage: ruby-build [-kpv46] <definition> <prefix>
>
> I wouldn't add further documentation for -46 because those are flags that
> I've never approved and am not sure what are they even used for. #599
> (comment)
> <https://github.com/rbenv/ruby-build/pull/599#issuecomment-151804316>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rbenv/ruby-build/pull/879/files#r49930410>.
>